src/util.cc

```diff
@@ -622,3 +624,48 @@ bool Truncate(const string& path, size_t size, string* err) {
   }
   return true;
 }
+
+/// Replace the content of file_dst with the content of new_content.
+/// The file pointed to by file_dst will keep the same uid and gid.
+/// In case of an error, false is returned and *err will carry the error message.
+bool ReplaceContent(const string& file_dst, const string& new_content, string* err) {
+#ifdef _WIN32
+  // This will not carry over uid / gid stats
+  // However, this is only in windows, on linux uid / gid will be preserved
```

Comment:
You can remove those two comments as it is now mentioned in the docstring of the function.